mon-pix/translations/fr.json

```
@@ -1,10 +1,62 @@
 {
+  "api-error-messages": {
+    "login-unauthorized-error": "L'adresse e-mail ou l'identifiant et/ou le mot de passe saisis sont incorrects.",
+    "bad-request-error": "Les données que vous avez soumises ne sont pas au bon format.",
+    "internal-server-error": "Une erreur interne est survenue, nos équipes sont en train de résoudre le problème. Veuillez réessayer ultérieurement."
+  },
+
+  "certificate": {
+    "back-link": "Retour à mes certifications",
+    "candidate-birth-complete": "Né(e) le {birthdate} à {birthplace}",
+    "certification-center": "Centre de certification :",
+    "clea-title": "Certifié Cléa",
+    "competences": {
+      "informations": "Votre score certifié a été calculé en fonction de vos réponses lors du passage de la certification. Il peut donc différer du nombre de Pix affiché dans votre profil. Le jour de votre certification il était possible d’atteindre le niveau 5 des compétences et 640 pix au maximum.",
+      "level": "niveau",
+      "sub-title": "(niveaux sur 5)",
```

Je crois que `subtitle` s'écrit en un mot:

```suggestion
      "subtitle": "(niveaux sur 5)",
```